demos/kubernetes/grafana.yaml

```diff
@@ -39,6 +39,8 @@ spec:
           value: Admin
         - name: GF_PATHS_DATA
           value: /data/grafana
+        - name: DASHBOARD_URL
+          value: https://raw.githubusercontent.com/istio/mixer/master/deploy/kube/conf/grafana-dashboard.json
         volumeMounts:
```

Comment:
cant we keep a copy of this in this repo? master branch version might change, but shouldnt break the demo.Unless this file is very static